src/Util/AdminSubjectExtractor.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Util;
```

Comment:
No `Util` namespace please, it's going to get cluttered with unrelated stuff, I guarantee it.

Reply:
I completely agree. I initially choose this NS based on the simility with `ClassUtils::getClass()`. When we have matured the scope of these responsibilities, I think we could choose a better name.